src/object/object_print.c

```diff
@@ -385,24 +384,15 @@ help_class_names (const char *qualifier)
 	      owner = db_get_owner (m->op);
 	      if (!requested_owner || ws_is_same_object (requested_owner, owner))
 		{
-		  cname = db_get_class_name (m->op);
+		  unique_name = db_get_class_name (m->op);
 		  buffer[0] = '\0';
-		  if (!requested_owner && db_get (owner, "name", &owner_name) >= 0)
+		  if (!requested_owner)
 		    {
-		      tmp = db_get_string (&owner_name);
-		      if (tmp)
-			{
-			  snprintf (buffer, sizeof (buffer) - 1, "%s.%s", tmp, sm_remove_qualifier_name (cname));
-			}
-		      else
-			{
-			  snprintf (buffer, sizeof (buffer) - 1, "%s.%s", "unknown_user", cname);
-			}
-		      db_value_clear (&owner_name);
+		      snprintf (buffer, sizeof (buffer) - 1, "%s", unique_name);
 		    }
 		  else
 		    {
-		      snprintf (buffer, sizeof (buffer) - 1, "%s", cname);
+		      snprintf (buffer, sizeof (buffer) - 1, "%s", sm_remove_qualifier_name (unique_name));
```

Comment:
What If `sm_remove_qualifier_name` returns NULL?

Reply:
Since the sm_remove_qualifier_name() function returns a const char * type,
how about changing it like the one below?

```
if (name == NULL || name[0] == '\0')
  {
    ERROR_SET_WARNING_1ARG (error, ER_SM_INVALID_NAME, name);
    return "";
  }
```

Reply:
Thank you.
Added exception handling code for returning NULL.